pages/common/ssh-keygen.md

```diff
@@ -7,13 +7,9 @@
 
 `ssh-keygen`
 
-- Specify file in which to save the key:
+- Generate an ed25519 key with 100 key derivation function rounds and specify the file in which to save the key:
```

Comment:
I think this one is easier to read.
```suggestion
- Generate an ed25519 key with 100 key derivation function rounds and save the key to a file:
```

Reply:
```suggestion
- Generate an ed25519 key with 100 key derivation function rounds and save the key to a specific file:
```
How about this?

Reply:
I'm more keen blueskuson's versions here.

Refer to: https://github.com/tldr-pages/tldr/pull/9874#discussion_r1162514532